Makefile.am

```diff
@@ -61,7 +61,7 @@ OSX_PACKAGING = $(OSX_DEPLOY_SCRIPT) $(OSX_FANCY_PLIST) $(OSX_INSTALLER_ICONS) \
 
 COVERAGE_INFO = baseline_filtered_combined.info baseline.info \
   leveldb_baseline.info test_dash_filtered.info total_coverage.info \
-  baseline_filtered.info rpc_test.info rpc_test_filtered.info \
+  baseline_filtered.info functional_test.info functional_test_filtered.info \
```

Comment:
missing changes in `.travis.yml`, should update matrix.h(?)

Reply:
same, please clarify